ui/src/components/Bouts/FeedItem.tsx

```diff
@@ -32,7 +32,10 @@ export default function FeedItem({
   feed: Pick<Feed, "id" | "name" | "slug" | "online">;
   onStatUpdate?: (feedId: string, stat: string, value: number) => void;
 }) {
-  const categories: Array<DetectionCategory> = ["WHALE", "VESSEL", "OTHER"];
+  const categories: Array<DetectionCategory> = useMemo(
```

Comment:
Doesn't need useMemo, this can just be [moved outside the component](https://react.dev/learn/removing-effect-dependencies#move-static-objects-and-functions-outside-your-component) (and then you can remove it from the useEffect dependency array)